package.json

```diff
@@ -57,7 +57,7 @@
     "ember-moment": "4.2.1",
     "ember-notify": "4.3.1",
     "ember-responsive": "1.2.3",
-    "ivy-codemirror": "~1.2.0",
+    "ivy-codemirror": "piotrpalek/ivy-codemirror#pass-params",
```

Comment:
I've made a [PR](https://github.com/IvyApp/ivy-codemirror/pull/12) to CodeMirror